src/checks_gen.rs

```diff
@@ -83,6 +83,9 @@ pub enum CheckCodePrefix {
     C415,
     C416,
     C417,
+    C9,
+    C90,
+    C901,
```

Comment:
It's confusing that both `flake8-comprehensions` and `mccabe` are now enabled by `--select C`, but I guess that's already true in Flake8?

Reply:
@charliermarsh yeah, this is true in flake8. We could use a different letter too. I know bandit uses `B` but `flake8-bandit` uses `S` to avoid collisions.

Reply:
I think it's ok, I'd rather be compatible than try to rename check codes.